### PR TITLE
Position move time check

### PR DESCRIPTION
**Pull request by randaz81**

### CODE REVIEW

File: src/libraries/icubmod/embObjMotionControl/embObjMotionControl.cpp [hunk 1]
```diff
@@ -2347,6 +2349,13 @@ bool embObjMotionControl::setPositionModeRaw()
 
 bool embObjMotionControl::positionMoveRaw(int j, double ref)
 {
+    if (_last_position_move_time[j] == 0) _last_position_move_time[j] = yarp::os::Time::now()-1.0;
```
Comment (lornat75) [inline src/libraries/icubmod/embObjMotionControl/embObjMotionControl.cpp]: Why this?

**Comment by lornat75** [comment]
Please wait some time before merging pull requests that are non-trivial like this one!

**Comment by randaz81** [comment]
Well, actually it's trivial... It's just a warning message. Before merging, it was extensively tested on both CAN and ETH architecture. RobotInterfaces now prints a warning (but still accepts the command) if the user misuses positionMove(). (the time interval between two position commands should be >80ms)

**Comment by randaz81** [comment]
The highlighted line of code was removed in master. I was not able to push the commit before. Today github server has some network problems.

**Pull request merged**